docs/anonymizer/adding_operators.md

```diff
@@ -0,0 +1,18 @@
+# Supporting new types of PII operators
```

Comment:
Maybe add a clarification for what an operator is? Something like "Operator: a method for either anonymizing or deanonymizing a PII entity. Example operators: Replace, Mask, Encrypt, Decrypt."

Reply:
Done.